test/edu-benefits/1990e/config/educationHistory.unit.spec.jsx

```diff
@@ -19,7 +19,7 @@ describe('Edu 1990e educationHistory', () => {
           definitions={definitions}/>
     );
     const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
```

Comment:
No, it should use getFormDOM, I just hadn't updated it because there are only `input` elements on this page. I should update it, though.